### DISCUSSION TITLE
狗狗币每次使用后需要备份文件吗，不是备用了秘钥就可以直接登录吗？

### DISCUSSION THREAD

**Discussion by yp7622266** [dogecoin/dogecoin#3706] [General]
狗狗币每次使用后需要备份文件吗，不是备用了秘钥就可以直接登录吗？ 如果每次都要备份文件 那就太麻烦了

**Comment by patricklodder** [comment]
If you have a wallet created with BIP32 functionality (i.e. on 1.14.x) then you don't need to back up after every use and you can restore the entire wallet from your initial back-up, unless you import keys manually.

**Reply by yp7622266** [reply]
我只保存秘钥，不备份电脑文件。  很多年后没有电脑文件，只有秘钥可以找回吗
